Source/Core/Core/Boot/Boot.h

```diff
@@ -26,10 +26,10 @@ class IOFile;
 
 struct RegionSetting
 {
-  const std::string area;
-  const std::string video;
-  const std::string game;
-  const std::string code;
+  char area[4];
```

Comment:
With how `CBoot::EmulatedBS2_GC` uses this structure, won't using `const char*` for those be "cleaner"? I'd probably be a bit paranoid about those fixed array sizes which are non-issue with a pointer type.